docs-requirements.txt

```diff
@@ -0,0 +1,2 @@
+sphinx>=1.6.2,<1.6.6
```

Comment:
this should be an easy moban template to add upstream, or it might even be an asset as all repos should have the same docs-requirements...

Reply:
Created an issue https://gitlab.com/coala/mobans/issues/36

Reply:
It was assigned to you , and I replied to your query there